debian/pbspro-server.install

```diff
@@ -0,0 +1,308 @@
+etc/pbs_db_schema.sql /opt/pbs/etc
```

Comment:
That all said, it is still concerning that we need to add so many files - and maintaining them will not be easy.
I suggest we break this ticket into two parts.
1. Allow PBS code to compile on Ubuntu
2. Package on Ubuntu. 

For the 1st ticket we can submit the PBS code changes required to build PBS on Ubuntu. 
For the 2nd ticket, I think we should try out @mike0042 approach of using Alien.